Back-end/src/com/smartplanner/BruteForcer.java

```diff
@@ -0,0 +1,75 @@
+package com.smartplanner;
+
+import java.util.ArrayList;
+
+
+//TODO: make cleanup
+public class BruteForcer
+{
+    private ArrayList<Activity> activities;
+    private int termIndexes[];
+    private boolean hasFinished=false;
+    private int maxDaysInCycle;
+
+    public BruteForcer(ArrayList<Activity> activities, int maxDaysInCycle)
+    {
+        this.maxDaysInCycle=maxDaysInCycle;
+        this.activities=activities;
+        this.termIndexes = new int[activities.size()];
+
+        for(int index : termIndexes)
+            index=0;
+    }
+
+    public boolean isNext()
+    {
```

Comment:
sometimes you put "{" in new line and sometimes not. You should use one convention